aptos-move/framework/aptos-framework/sources/Token.move

```diff
@@ -151,48 +179,56 @@ module AptosFramework::Token {
     }
 
     public(script) fun create_limited_token_script(
-        account: &signer,
+        creator: &signer,
         collection: vector<u8>,
         name: vector<u8>,
         description: vector<u8>,
         monitor_supply: bool,
         initial_balance: u64,
         maximum: u64,
         uri: vector<u8>,
+        custom_metadata: Table<vector<u8>, vector<u8>>,
```

Comment:
offline discussed with @msmouse , this is not doable. To support customized attributes, will need a different way. currently just use a json blob